### PR TITLE
Jay Wingler Fluff Item

### PR DESCRIPTION
**Pull request by DaveTheHeadcrab**
Requested by Jayfeather.
A comb that turns a Tajaran's tail into one with markings on it.
Icons are as follows:
comb
tajnudes2
tajnudes1

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -171,6 +173,28 @@
...
+	name = "blue comb"
+	desc = "A blue comb, it looks like it was made to groom a Tajaran's fur."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "wingler_comb"
+	attack_verb = "combed"
+	force = 0
+	throwforce = 0
+	var/used = 0
+
+/obj/item/fluff/wingler_comb/attack_self(mob/user as mob)
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: `/obj/item/fluff/wingler_comb/attack_self(mob/user)`

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -171,6 +173,28 @@
...
+	if(used)
+		return
+
+	var/mob/living/carbon/human/target = user
+	if(!istype(target) || !istype(target.species, /datum/species/tajaran)) // Only catbeasts, kthnx.
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: Please use the the proper species proc for returning the species--ideally, everything should use those procs to check for species and not istype checking them (especially with @Tastyfish's eventually non-global species datumization).

File: code/modules/customitems/item_defines.dm [hunk 3]
```diff
@@ -171,6 +173,28 @@
...
+		return
+
+	if(target.change_body_accessory("Jay Wingler Tail"))
+		to_chat(target, "<span class = 'notice'>You comb your tail with the [src].</span>")
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: `to_chat(target, "<span class='notice'>You comb your tail with the [src].</span>")`

**Comment by Fox-McCloud** [comment]
@Necaladun

**Comment by Dumbdumn5** [comment]
BALANCE BREAKING OMG WTF!!!!!1!111 :-1: :x: :100:

**Comment by KasparoVy** [comment]
Good call making it a body accessory.

**Comment by DaveTheHeadcrab** [comment]
The comb is a recolor from an old bay fluff item.
I did the tail sprite work.

**Pull request merged**